File: Core/Workspace/Typescript.legacy/Site/wwwroot/Core/Workspace/Protocol/Invoke/InvokeRequest.ts
namespace Allors.Protocol {

    export interface InvokeRequest {
        i: Invocation[];
        o?: InvokeOptions;
    }

    export interface Invocation {
        i: string;
        v: string;
        m: string;
    }

    export interface InvokeOptions {
        // Isolated
        i?: boolean;

        // ContinueOnError
        c?: boolean;
    }
}

File: dotnet/platform/protocol/allors.protocol/api/invoke/invokerequest.cs
// <copyright file="InvokeRequest.cs" company="Allors bvba">
// Copyright (c) Allors bvba. All rights reserved.
// Licensed under the LGPL license. See LICENSE file in the project root for full license information.
// </copyright>

namespace Allors.Protocol.Remote.Invoke
{
    public class InvokeRequest
    {
        /// <summary>
        /// Invocations
        /// </summary>
        public Invocation[] i { get; set; }

        /// <summary>
        /// Invoke Options
        /// </summary>
        public InvokeOptions o { get; set; }
    }
}
